presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorCommitHandle.java

```diff
@@ -13,6 +13,16 @@
  */
 package com.facebook.presto.spi.connector;
 
+import com.facebook.presto.spi.SchemaTableName;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+import java.util.Collections;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "@class")
```

Comment:
Hi @highker, the reason Jackson complains is they do not know the type of the connectorcommithandle interface during the deserialization. After adding this type info into the serialization, Jackson stops complaining. I have run verifier test, and it works (so I think we pass connectorcommithandle to output object) . Since it covers up all the details from presto-main, it might be the best way we can have. What do you think?